scripts/uberenv/packages/serac/package.py

```diff
@@ -88,6 +88,8 @@ class Serac(Package):
     # Basic dependencies
     depends_on("mpi")
 
+    depends_on("axom~fortran~raja~umpire")
```

Comment:
Why are we not compiling raja and umpire? We obviously don't need it, but is there a deeper problem that we're avoiding?

Reply:
We don't currently need it.

Reply:
Specifying variant because the feature is not needed while it wouldn’t break the code puts an unnecessary constraint on the concretization. Please don’t do that...
Typically, it will force to install a specific version of axom, and it will make packaging of mini-apps together more difficult.

Reply:
Having a variant for CUDA (which would turn on raja and umpire) will eventually be needed but we don't need it right now.

Also we currently expect people to either use our library builds or use our library build process, which builds the whole thing.

When Serac is ready for prime time we can work on removing these constraints and making the Serac Spack package more general purpose.  I don't want to hold up progress fixing continuous build issues and building libraries I am not currently using, which I have spent entirely too much time lately doing.

Reply:
I understand, I just had the same issue with conduit for a similar choice made 4 years ago.
I’m pretty sure you could still specify the default variants you want for axom in pakages.yaml though. It seems to be an interesting compromise, don’t you think?

Reply:
That seems like a reasonable compromise, although I am admittedly not a spack expert (or even sensible user).